iwyu_path_util.cc

```diff
@@ -178,9 +177,13 @@ string GetParentPath(const string& path) {
 
 // Converts a file-path, such as /usr/include/stdio.h, to a
 // quoted include, such as <stdio.h>.
-string ConvertToQuotedInclude(const string& filepath) {
+string ConvertToQuotedInclude(const string& filepath, const string& includerPath) {
```

Comment:
You mean `includerPath` -> `includer_filepath`? So no CamelCasing for variables?